otx/algorithms/segmentation/configs/resnet50_deeplabv3plus/template.yaml

```diff
@@ -0,0 +1,61 @@
+# Description.
+model_template_id: Custom_Semantic_Segmentation_ResNet50_ASPP
```

Comment:
confusing name.. deeplabv3plus_resnet50? or aspp_resent50?

Reply:
Like I commented above, I think we need to align the name of template and file name.